packages/snap-examples/examples/bls-signer/index.html

```diff
@@ -35,8 +33,7 @@ <h1>BLS Signature Snap Example!</h1>
   </body>
 
   <script>
-
-    const snapId = new URL('package.json', window.location.href).toString()
+    const snapId = `local:${window.location.href}`;
```

Comment:
I'm not too fond of having source (`local`) be a part of the snap id

Reply:
This was discussed / resolved out of band.